src/Components/test/E2ETest/Tests/BootResourceCachingTest.cs

```diff
@@ -93,12 +92,12 @@ public void IncrementallyUpdatesCache()
             requestedDll => Assert.Contains("/dotnet.native.wasm", requestedDll));
 
         // We also update the cache (add new items, remove unnecessary items)
+        await Task.Delay(10500); // wait for cache purge, which is 10 seconds delayed after app start
```

Comment:
Does this always take 10 seconds, or can we optimistically check the cache earlier (maybe once every 100 ms or something) and only fail if the cache does not update after 10.5 seconds? I hate unnecessarily long tests.

If that's just how long it usually takes now, can we reconfigure it to be shorter just for the purposes of this test. Giving only 10.5 seconds for something that normally takes 10 seconds seems like a pretty tight window that's likely to lead to flakiness anyway.

Reply:
Delaying cache purge by 10 seconds is recent change in the runtime repo I made, so that it doesn't compete with startup performance. It's arbitrary and not yet configurable. But it can be made configurable.

Reply:
https://github.com/dotnet/runtime/pull/90264

Reply:
I will improve that test when the change flows here, it could be next dependency PR